diff --git a/dbus-x11_1.12.2-1ubuntu1.1_amd64/DEBIAN/conffiles b/dbus-x11_1.12.2-1ubuntu1.2~16.04.sav0_amd64/DEBIAN/conffiles
index 0c81010..b414f79 100644
--- a/dbus-x11_1.12.2-1ubuntu1.1_amd64/DEBIAN/conffiles
+++ b/dbus-x11_1.12.2-1ubuntu1.2~16.04.sav0_amd64/DEBIAN/conffiles
@@ -1,2 +1,3 @@
 /etc/X11/Xsession.d/75dbus_dbus-launch
 /etc/X11/Xsession.d/95dbus_update-activation-env
+/etc/X11/Xsession.d/99vw_dbus_upd-activation-env
diff --git a/dbus-x11_1.12.2-1ubuntu1.1_amd64/DEBIAN/control b/dbus-x11_1.12.2-1ubuntu1.1~16.04.sav0_amd64/DEBIAN/control
index 1718d22..3900bfa 100644
--- a/dbus-x11_1.12.2-1ubuntu1.1_amd64/DEBIAN/control
+++ b/dbus-x11_1.12.2-1ubuntu1.2~16.04.sav0_amd64/DEBIAN/control
@@ -1,6 +1,6 @@
 Package: dbus-x11
 Source: dbus
-Version: 1.12.2-1ubuntu1.1
+Version: 1.12.2-1ubuntu1.2~16.04.sav0
 Architecture: amd64
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Installed-Size: 147
diff --git a/dbus-x11_1.12.2-1ubuntu1.1_amd64/etc/X11/Xsession.d/99vw_dbus_upd-activation-env b/dbus-x11_1.12.2-1ubuntu1.2~16.04.sav0_amd64/etc/X11/Xsession.d/99vw_dbus_upd-activation-env
new file mode 100644
index 0000000..1626e74
--- /dev/null
+++ a/dbus-x11_1.12.2-1ubuntu1.1_amd64/etc/X11/Xsession.d/99vw_dbus_upd-activation-env
@@ -0,0 +1,17 @@
+# vim:set ft=sh sw=2 sts=2 et:
+
+if [ -n "$DBUS_SESSION_BUS_ADDRESS" ] && \
+    [ -x "/usr/bin/dbus-update-activation-environment" ]; then
+  # subshell so we can unset environment variables
+  (
+    # unset login-session-specifics
+    unset XDG_SEAT
+    unset XDG_SESSION_ID
+    unset XDG_VTNR
+
+    # tell dbus-daemon --session (and systemd --user, if running)
+    # to put the Xsession's environment in activated services'
+    # environments
+    dbus-update-activation-environment --verbose --systemd --all
+  )
+fi
